src/main/java/seedu/modtracker/Module.java

```diff
@@ -51,4 +55,22 @@ public void minusActualTime(String time, String week) {
         int i = Integer.parseInt(week);
         this.actualTime[i] -= d;
     }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj) {
+            return true;
+        }
+        if (!(obj instanceof Module)) {
+            return false;
+        }
+        Module m = (Module) obj;
+
+        return moduleCode.equals(m.moduleCode);
+    }
```

Comment:
Why not just take in Module as an argument?

Reply:
The default `Object.equals()` method takes in the object of Object type as the parameter and I keep this method signature to overwrite the default method.